test/va-letters/reducers/index.unit.spec.js

```diff
@@ -0,0 +1,63 @@
+import { expect } from 'chai';
+
+import lettersReducer from '../../../src/js/va-letters/reducers';
+
+const initialState = {
+  letters: [],
+  destination: {},
+  available: false
+};
+
+describe('letters reducer', () => {
+  it('should handle failure to fetch letters', () => {
+    const state = lettersReducer.letters(
```

Comment:
Yeah, this is due to how the common store is set up. When `va-letters/reducer` is pulled in by the common store function, it's wrapped in a `combineReducers` call, which turns that object into a function. Since you're importing the reducer directly, you're skipping that step. We used to have the `combineReducers` call in the reducer code for each app, but we can't do that and easily combine it with the user profile/login reducers.